src/e2e_tests/sample_scripts/test_very_parametrized_script.py

```diff
@@ -253,3 +254,34 @@ def test_check_search_results_in_command_based_list(browser, config_host):
         expect(str(element.text).find(search_request) > -1)
 
     assert_expectations()
+
+
+@severity(severity_level.NORMAL)
+@allure.title("Check Multiple selection")
+def test_check_multiple_selection(browser, config_host):
+    very_parametrized_script_page = VeryParametrizedScript(browser, config_host)
+    very_parametrized_script_page.parameter_multiple_selection.click()
+
+    expect(is_displayed(very_parametrized_script_page.parameter_multiple_selection_drop_down), "Drop down for Multiple selection not found after click")
+    assert_expectations()
+
+
+@severity(severity_level.NORMAL)
+@allure.title("Check Multiple selection check/uncheck action")
+def test_check_multiple_selection_check_uncheck_action(browser, config_host):
+    very_parametrized_script_page = VeryParametrizedScript(browser, config_host)
+
+    if len(very_parametrized_script_page.parameter_multiple_selection_drop_down_checked_elements) > 0:
```

Comment:
Since you are working with predefined scripts, I think this `if` is redundant here. You should know what to expect there.